egs/wsj/s5/steps/nnet3/report/generate_plots.py

```diff
@@ -478,7 +481,7 @@ def Main():
     GeneratePlots(args.exp_dir, args.output_dir,
                   comparison_dir = args.comparison_dir,
                   start_iter = args.start_iter,
-                  is_chain = args.is_chain)
+                  objective_type = args.objective_type)
 
 if __name__ == "__main__":
     Main()
```

Comment:
Vimal, I think the module loading in this file should also be changed, `train_lib = imp.load_source('ntl', 'steps/nnet3/nnet3_train_lib.py')` is deleted in your PR